test/client/fixtures/utils/event-obj-wrapper-test.js

```diff
@@ -0,0 +1,75 @@
+var iframeSandbox = hammerhead.sandbox.iframe;
+var nativeMethods = hammerhead.nativeMethods;
+
+QUnit.testStart(function () {
+    iframeSandbox.on(iframeSandbox.RUN_TASK_SCRIPT_EVENT, initIframeTestHandler);
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT_EVENT, iframeSandbox.iframeReadyToInitHandler);
+});
+
+QUnit.testDone(function () {
+    iframeSandbox.off(iframeSandbox.RUN_TASK_SCRIPT_EVENT, initIframeTestHandler);
+});
+
+function getOwnProperties (obj) {
+    var props = [];
+
+    for (var prop in obj) {
+        if (obj.hasOwnProperty(prop))
+            props.push(prop);
+    }
+
+    return props;
+}
+
+module('regression');
+
+test('the "message" event object should be correctly overridden (GH-1277)', function () {
```

Comment:
Let's split to two tests:
`test('the overriden "message" event object should have same own properties as native (GH-1277)'

`test('the overriden "message" event object should stringified as native (GH-1277)'